libsyclinterface/source/dpctl_sycl_platform_interface.cpp

```diff
@@ -202,3 +203,18 @@ __dpctl_give DPCTLPlatformVectorRef DPCTLPlatform_GetPlatforms()
     // the wrap function is defined inside dpctl_vector_templ.cpp
     return wrap(Platforms);
 }
+
+__dpctl_give DPCTLSyclContextRef
+DPCTLPlatform_GetDefaultContext(__dpctl_keep const DPCTLSyclPlatformRef PRef)
+{
+    auto P = unwrap(PRef);
+    if (P) {
+        auto default_ctx = P->ext_oneapi_get_default_context();
+        return wrap(new context(default_ctx));
+    }
+    else {
+        error_handler("Driver version cannot be looked up for a NULL platform.",
```

Comment:
Hmm what is this error message? Copy pasta?

Reply:
Right. Sorry.